cli/submit.py

```diff
@@ -54,7 +69,7 @@ def auth_prompt(self):
         choice = int(input("Enter the number corresponding to your choice: "))
 
         if choice == 1:
```

Comment:
I think we'll have to review how we offer a choice to our users. 
This will prevent any unmonitored submission.

Reply:
Created EVA-3339